content/managing-apps/installing-custom-apps.md

```diff
@@ -0,0 +1,1280 @@
+
+Replace it with the absolute path to the Dockerfile, for example:
+
+{{< codeblock language="yaml" >}}nextcloud:
+    build:
+      context: .
+      dockerfile: /mnt/virtual/dockerfiles/nextcloud.dockerfile{{< /codeblock >}}
+
+Note: an application with an absolute path to a Dockerfile is not portable without modification.
+
+#### Installing Multiple Apps on a Shared Custom Network
+These examples deploy two applications, Plex and Tautulli, as part of the same custom Docker [network](https://docs.docker.com/engine/network/).
+Grouping Docker applications offers advantages for organizing, securing, and optimizing communication between containers that interact.
+You can use Docker networks to isolate network segmentation, simplify communication between apps, configure networking properties, and more.
+
+**Plex**
+
+{{< codeblock language="yaml" >}}networks:
+  plex_net:
+    name: plex_net
+    driver: bridge
+    external: false
+
+services:
+  plex:
+    networks:
+     - plex_net
+    image: ghcr.io/onedr0p/plex:1.41.0.8994-f2c27da23@sha256:fe9ed1189617cd98106635c63ed39d40e4ec82535514e4a51a7c72e56b65c4dd
+    restart: always
+    user: "UID:GID" # UPDATE ME -- MAKE SURE USER HAS ACL PERMISSIONS FOR ALL STORAGE PATHS BELOW
+    environment:
+      NVIDIA_VISIBLE_DEVICES: void
+      # https://support.plex.tv/articles/201105343-advanced-hidden-server-settings/
+      PLEX_PREFERENCE_secureConnections: "secureConnections=0"
+      PLEX_PREFERENCE_LanNetworksBandwidth: "LanNetworksBandwidth=172.16.0.0/12,10.0.0.0/8,192.168.0.0/16"
+      PLEX_PREFERENCE_customConnections: "customConnections=https://10.20.30.254:32400"
+    ports:
+      - 32400:32400
+    volumes:
+      - /mnt/cache/appsdata/plex:/config
+      - /mnt/virtual/library/tv:/tv:ro #UPDATE LOCAL PATH
+      - /mnt/virtual/library/movies:/movies:ro #UPDATE LOCAL PATH
+    tmpfs:
+      - /transcode
+    healthcheck:
+      test: 'curl --silent --fail http://localhost:32400/identity'
+      interval: 15s
+      timeout: 5s
+      retries: 15
+      start_period: 60s{{< /codeblock >}}
+
+**Tautulli**
+
+{{< codeblock language="yaml" >}}networks:
+  plex_net:
+    name: plex_net
+    driver: bridge
+    external: true
+
+services:
+  tautulli:
+    image: tautulli/tautulli:v2.14.5@sha256:6017b491d8e9100a97391b639fff5824ad36a315c69aae3c9ed78407994a626e
+    restart: always
+    entrypoint:
+      - python
+      - Tautulli.py
+    command:
+      - --port
+      - "8989"
+      - --config
+      - /config/config.ini
+      - --datadir
+      - /config
+    networks:
+      - plex_net
+    volumes:
+      - /mnt/cache/appdata/tautulli:/config
+      - /mnt/cache/appdata/plex/Library/Application Support/Plex Media Server/Logs:/plexlogs:ro
+    ports:
+      8989:8989
+    healthcheck:
+      test: curl --silent --fail http://localhost:8989/status
+      interval: 10s
+      timeout: 5s
+      retries: 5
+      start_period: 30s{{< /codeblock >}}
+
+## Managing Custom Apps
```

Comment:
Would this section, if it is really straightforward and not different enough from managing a non-custom app, be better served from the managing-installed-apps article?